app/upgrades/v18/upgrades.go

```diff
@@ -1,28 +0,0 @@
-package v18
```

Comment:
Note to reviewer: We just need to replace this with the actual upgrade handler we used in v18 (was never ported from feature branch to main)